packages/edit-site/src/components/sidebar-navigation-root/style.scss

```diff
@@ -1,4 +1,3 @@
 .edit-site-sidebar-navigation-root {
-	/* 14px corresponds to the button padding and the grid gap of the title */
-	margin-left: $button-size - 4px;
+	margin: 0 $grid-unit-50
```

Comment:
I think this should still be dependent on "button-size" which is the size of the "back" button and 4px corresponds the spacing you just modified, so the correct result now would be `$button-size + 8px` (since you doubled the spacing)

Reply:
The value needs to be 40px to visually align everything along the Y axis. I can use `$button-size` to achieve that though 👍